packages/backend/src/analytics/LightdashAnalytics.ts

```diff
@@ -1056,6 +1057,38 @@ export type SemanticLayerView = BaseTrack & {
     };
 };
 
+export type VirtualViewCreatedEvent = BaseTrack & {
+    event: 'virtual_view.created';
```

Comment:
should we unify all these events and make `name` optional ?

Reply:
done!